image/scanner/scripts/import-additional-cas

```diff
@@ -1,8 +1,17 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 set -euo pipefail
 
-[ -d /usr/local/share/ca-certificates ] || exit 0
-[ "$(find /usr/local/share/ca-certificates -maxdepth 1 -name '*.crt' | wc -l)" -gt 0 ] || exit 0
-cp -L /usr/local/share/ca-certificates/* /etc/pki/ca-trust/source/anchors
+copy_existing () {
+    src=$1
+    if [ -d "$src" ] && [ "$(ls -A -I "..*" "$src")" ]; then
+        cp -v -L "$src"/* /etc/pki/ca-trust/source/anchors
```

Comment:
On one hand I like `-v`, on the other hand, this code is called each time the image is started and will produce some logging that may be noisy. WDYT?

Reply:
Feel free to ignore for consistency with https://github.com/stackrox/stackrox/pull/1088

Reply:
That `-v` was added specifically for generating the little noise for logging purposes to simplify debugging.